content/en/docs/tasks/configure-pod-container/configure-pod-initialization.md

```diff
@@ -80,6 +80,11 @@ The output shows that nginx is serving the web page that was written by the init
       ...
 
 
+Exit the shell.
```

Comment:
```suggestion
```

I'd recommend removing this since the next step requires you use a shell/shell emulator to execute.